core/vm/eips.go

```diff
@@ -280,3 +281,19 @@ func enable7702(jt *JumpTable) {
 	jt[STATICCALL].dynamicGas = gasStaticCallEIP7702
 	jt[DELEGATECALL].dynamicGas = gasDelegateCallEIP7702
 }
+
+// opCLZ implements the CLZ opcode (count leading zero bytes)
+func opCLZ(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
+	x := scope.Stack.peek()
+	x.SetUint64(256 - uint64(x.BitLen()))
+	return nil, nil
+}
+
+func enable7939(jt *JumpTable) {
+	jt[CLZ] = &operation{
+		execute:     opCLZ,
+		constantGas: GasFastestStep,
```

Comment:
This is incorrect, please apply https://github.com/ethereum/go-ethereum/pull/32172/files

Reply:
updated. there is another pr https://github.com/ethereum/go-ethereum/pull/32352, but I think we can still keep use `interpreter *EVMInterpreter`